scene/gui/tree.cpp

```diff
@@ -5708,6 +5708,10 @@ Variant Tree::get_drag_data(const Point2 &p_point) {
 }
 
 TreeItem *Tree::get_item_at_position(const Point2 &p_pos) const {
+	if (!Rect2(get_position(), get_size()).has_point(p_pos)) {
```

Comment:
AFAICT this method works in local coords, so the `position` is irrelevant here.

```suggestion
	if (!Rect2(Point2(), get_size()).has_point(p_pos)) {
```


Besides, there are other `Tree::get_{...}_at_position` methods having similar duplicated logic (and thus probably the same issue). Would make sense to refactor them so they'd have shared finding logic. But as a simple fix adding similar checks to them should do. :thinking: